### PR TITLE
Calculate area for report based on configured start/end lat/long.

### PR DESCRIPTION
**Pull request by pdong**
Added some code to calculate the scraped area based on app configs.

### CODE REVIEW

File: web.py [hunk 1]
```diff
@@ -197,11 +199,25 @@ def report_main():
     }
     session_stats = db.get_session_stats(session)
     session.close()
+
+    lat1 = app_config.MAP_START[0]
```
Comment (modrzew) [inline web.py]: Could you move that to `utils.py` as a separate function?

**Comment by pdong** [comment]
Yup.  moved over to utils as get_scan_area()

File: utils.py [hunk 2]
```diff
@@ -9,6 +11,21 @@ def get_map_center():
...
+    lon2 = config.MAP_END[1]
+    p1 = Point(lat1, lon1)
+    p2 = Point(lat1, lon2)
+    p3 = Point(lat1, lon1)
+    p4 = Point(lat2, lon1)
+
+    width = distance.distance(p1,p2).kilometers
+    height = distance.distance(p3,p4).kilometers
+    area = int(width * height)
+    return area;
```
Comment (modrzew) [inline utils.py]: A **semicolon**?!

(sorry for late response, I found some time to process issues)

**Comment by pdong** [comment]
Haha whoops.  I pretty new to python; mainly do JavaScript and some old habits leaked over

**Comment by modrzew** [comment]
Thanks!

**Pull request merged**